src/browser/pages/vscode.ts

```diff
@@ -72,20 +72,22 @@ export function getNlsConfiguration(_document: Document, base: string) {
 
     type LoadBundleCallback = (_: undefined, result?: string) => void
 
-    nlsConfig.loadBundle = (bundle: string, _language: string, cb: LoadBundleCallback): void => {
+    nlsConfig.loadBundle = async (bundle: string, _language: string, cb: LoadBundleCallback): Promise<void> => {
       const result = bundles[bundle]
+
       if (result) {
         return cb(undefined, result)
       }
-      // FIXME: Only works if path separators are /.
```

Comment:
🧹 Removed this comment because it's related to `createBundlePath` and explained in a comment near that function with more details.